QueryBuilder/Dynamic/FilteredQuery.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DigitalWorkplace.DigitalTwins.QueryBuilder.Dynamic
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.DigitalWorkplace.DigitalTwins.QueryBuilder.Common.Clauses;
+    using Microsoft.DigitalWorkplace.DigitalTwins.QueryBuilder.Common.Helpers;
+    using Microsoft.DigitalWorkplace.DigitalTwins.QueryBuilder.Dynamic.Statements;
+
+    /// <summary>
+    /// A wrapper for the common functionalities of query builder.
+    /// </summary>
+    public class FilterQuery<TQuery, TWhereStatement> : DynamicQueryBase
```

Comment:
I actually changed this based on another suggestion you made. It would seem the correction here should be to update the file name, not the class name...